mongodbatlas/checkpoints.go

```diff
@@ -0,0 +1,91 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	backupCheckpoints = "groups/%s/clusters/%s/backupCheckpoints"
+)
+
+// CheckpointService is an interface for interfacing with the Checkpoint
+// endpoints of the MongoDB Atlas API.
+type CheckpointService interface {
+	List(context.Context, string, string, *ListOptions) (*Checkpoints, *Response, error)
+	Get(context.Context, string, string, string) (*Checkpoint, *Response, error)
+}
+
+//CloudProviderSnapshotRestoreJobsServiceOp handles communication with the CloudProviderSnapshotRestoreJobs related methods of the
+//MongoDB Atlas API
+type CheckpointsServiceOp struct {
+	Client RequestDoer
+}
+
+var _ CheckpointService = &CheckpointsServiceOp{}
+
+type Checkpoint struct {
```

Comment:
Thank you so much, it's really nice, but could you add comments describing all the importing functions and structs to have coherency? Also, it avoids warnings in some editors for missing comments.